mitodl/open-discussions
# CSAIL course import

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2738 

#### What's this PR do?
Imports CSAIL courses (all 3 of them 😉)
 
#### How should this be manually tested?
- Set `CSAIL_BASE_URL=https://cap.csail.mit.edu/` in your .env file
- Run `manage.py backpopulate_csail_data`
- Check the course search page, there should be 3 CSAIL courses
- Topic is hardcoded to "Computer Science" after discussing w/Peter.

#### Screenshots (if appropriate)
<img width="944" alt="Screen Shot 2020-04-09 at 11 25 35 AM" src="https://user-images.githubusercontent.com/187676/78928585-d95d8f00-7a6e-11ea-83da-008ccd176d0d.png">

<img width="476" alt="Screen Shot 2020-04-09 at 11 25 48 AM" src="https://user-images.githubusercontent.com/187676/78928596-e0849d00-7a6e-11ea-8bf1-e569b2a113e1.png">
